WebserverDockerfile

```diff
@@ -4,5 +4,5 @@ WORKDIR /app
 COPY package.json /app
 RUN npm install
 COPY . /app
-EXPOSE 80
-CMD ["node", "PlatePlannerServer.js"]
\ No newline at end of file
+EXPOSE 80 3000

```

Comment:
Adding another port is doable, but it might make more sense to have react on 80 and express on 3000? That way when people just go the URL they get the app, as opposed to the API.

Reply:
Just update, switching react to port 80 and express to port 3000

Reply:
Sweet!